### ISSUE TITLE
Load Datagridview  With Data From MySQL Database Using C#

### ISSUE THREAD

**Issue by tambastar** [section-engineering-education/engineering-education#1661]
### Introduction
In this article will learn about  Datagridview control and connecting it with the MySQL database. Datagridview is highly configurable and extensible, and it provides many properties, methods, and events to customize its appearance and behavior based on user requirements.
The tutorial will use the MySQL database as the main data source.

### Goals
1. Create a winform application
2. Create MySQL database
3. Connect C# application with MySQL database
4. Load data to the DatagridView 
5. Run DatagridView Events

### Prerequisites
- Prior C# Programming Language
- Prior MySQL Knowledge

###  Conclusion
In conclusion, the developer will learn how to directly connect to the MySQL database and perform several events within their application.
